src/components/tabController/TabBarItem.tsx

```diff
@@ -187,40 +179,63 @@ export default function TabBarItem({
     };
   });
 
+  const pressStyle = useAnimatedStyle(() => {
+    return {
+      backgroundColor: isPressed.value ? activeBackgroundColor : backgroundColor,
+      opacity: isPressed.value ? activeOpacity : 1
+    };
+  });
+
   const _style = useMemo(() => {
     const constantWidthStyle = itemWidth.current ? {flex: 0, width: itemWidth.current} : undefined;
-    return [styles.tabItem, style, constantWidthStyle];
+    return [styles.tabItem, style, constantWidthStyle, pressStyle];
   }, [style]);
 
+  const gesture = Gesture.Tap()
+    .maxDuration(60000)
```

Comment:
Is this limit solving a problem?

Reply:
Yes, it create a proper time between touch down and touch up. 
I didnt find any proper way to do it. 
I need to handle touch down - for the active press background color and opacity
and touch up - is for completing the actual press (and moving the current index)

Reply:
Oh, I just tested without it and with `.maxDuration(0)` and now I get it.
I'd expect it'll be possible to have unlimited time...